packages/react/src/components/Popup/Popup.tsx

```diff
@@ -122,7 +122,9 @@ export interface PopupState {
 }
 
 /**
- * A Popup displays additional information on top of a page.
+ * A Popup is a non-modal element that floats around its target on top of a page.
```

Comment:
Don't like 'floats around' - sounds like swimming.  A Popup is a non-modal element that is displayed above the target and is often used to display additional rich content relating to it's target.

Would be great to also describe concept.  The target is the element upon which an action caused the popup to be displayed....

Reply:
> Would be great to also describe concept. The target is the element upon which an action caused the popup to be displayed....

we have a test that checks that description is no longer than 25 words :) will try my best to do this